action.yml

```diff
@@ -4,6 +4,39 @@
   branding:
     icon: 'aperture'
     color: 'blue'
+  inputs:
+    installDir:
+      description: 'Installation folder of Parasoft SOAtest. If not specified, the soatestcli executable must be added to PATH.'
+      required: false
+    workingDir:
+      description: 'Working directory for running SOAtest.'
+      required: false
+      default: ${{ github.workspace }}
+    testConfig:
+      description: 'Test configuration to be used for test execution.'
+      required: false
+      default: 'user://Example Configuration'
+    resource:
+      description: 'Path to the test suite(s) relative to the workspace.'
+      required: false
+      default: 'TestAssets'
+    settings:
+      description: 'Path to a settings file relative to the workspace.'
+      required: false
+    reportDir:
+      description: 'Output folder for the test execution reports.'
+      required: false
+      default: 'reports'
+    reportFormat:
+      description: 'Format of test execution reports.'
+      required: false
+      default: 'xml,html'
+    environment:
+      description: 'Path to to a environment file relative to the workspace.'
```

Comment:
I think it's not path to a environment file but should be:
Name of the SOAtest environment to use for executing the tests.